fastparquet/writer.py

```diff
@@ -737,11 +788,19 @@ def write(filename, data, row_group_offsets=50000000,
         resolution; in "int96" mode, they are written as 12-byte blocks, with
         the first 8 bytes as ns within the day, the next 4 bytes the julian day.
         'int96' mode is included only for compatibility.
+    quantization_level: int or None
+        If not None, then quantization_level floating point values in this many units of the
+        median absolute deviation of the column. Only works for the 'simple' file
```

Comment:
Ohhhh I forgot. I only implemented this with the simple scheme. I will add HIVE too. I assumed that most libraries that want to read these files won't be able to since this is not in the parquet standard. Thus only enabling for simple files made sense to me.